### PR TITLE
NCO version fix for cemac arc4 

### PR DESCRIPTION
**Pull request by cemachelen**
* installed nco 4.6.0 on arc4 - default to this in config.bash
* added in a version check for nco main.bash
* if later versions are picked up then ncks will still extract chemistry variables
* added in a warning if chemvarlist or metvarlist are empty or the same so there's no secret cold starts

* fixed a link to user guide in the repo readme.md (was linking to Luke's own repository giving 404)

### CODE REVIEW

File: WRFotron2.0_CEMAC_ARC4/main_restart.bash [hunk 1]
```diff
@@ -59,7 +59,7 @@ done
 msg "chem"
```
Comment (lukeconibear) [inline WRFotron2.0_CEMAC_ARC4/main_restart.bash:60]: This file will also need the changes that were applied to main.bash.

File: WRFotron2.0_CEMAC_ARC4/namelist.chem.blueprint [hunk 2]
```diff
@@ -37,5 +37,5 @@
  have_bcs_chem                       = .true., .false., .false.,  ! gets lateral boundary data from wrfbdy (.true.) or idealized profile (.false.)
  have_bcs_upper                      = .false., .false., .false., ! upper boundary bounary condition for chemical species
  aer_op_opt                          = 2, 2, 2,                   ! aerosol optical properties, 1 = volume, 2 = approximate Maxwell-Garnet, 3 = complex volume-mixing, 4 = complex Maxwell-Garnet, 5 = complex core-shell
- bbinjectscheme                      = 2, 2, 2,                   ! 0 = plumerise (biomass_burn_opt), 1 = all ground level, 2 = flaming evenly in BL, 3 = flaming top BL, 4 = flaming injected at specific height
+ bbinjectscheme                      = 1, 1, 1,                   ! 0 = plumerise (biomass_burn_opt), 1 = all ground level (recommended), 2 = flaming evenly in BL, 3 = flaming top BL, 4 = flaming injected at specific height
```
Comment (lukeconibear) [inline WRFotron2.0_CEMAC_ARC4/namelist.chem.blueprint:40]: The recommended bbinjectscheme option is now 2. Please, could you change this back to 2?

File: WRFotron2.0_CEMAC_ARC4/namelist.wrf.blueprint [hunk 3]
```diff
@@ -16,23 +16,18 @@
  history_interval                    = 60, 60,                               ! history output file interval in minutes (integer only)
  frames_per_outfile                  = 1, 1,                                 ! number of output times bulked into each history file
  restart                             = __ISRESTART__,                        ! whether this run is a restart
- restart_interval                    = 1440,                                 ! restart output file interval in minutes
+ restart_interval                    = 360,                                 ! restart output file interval in minutes
```
Comment (lukeconibear) [inline WRFotron2.0_CEMAC_ARC4/namelist.wrf.blueprint:19]: The restart_interval should be greater than 360, i.e. we have used 1440. Please, could you change this back?

File: WRFotron2.0_CEMAC_ARC4/main_restart.bash (hunk 1 above)
Comment (cemachelen) [reply WRFotron2.0_CEMAC_ARC4/main_restart.bash]: That was silly of me to forget!

File: WRFotron2.0_CEMAC_ARC4/namelist.chem.blueprint (hunk 2 above)
Comment (cemachelen) [reply WRFotron2.0_CEMAC_ARC4/namelist.chem.blueprint]: done

File: WRFotron2.0_CEMAC_ARC4/namelist.wrf.blueprint [hunk 4]
```diff
@@ -43,8 +38,8 @@
  e_we                                = 140, 154,                             ! westeast dimension
  e_sn                                = 140, 154,                             ! southnorth dimension
  e_vert                              = 33, 33,                               ! vertical dimension
- num_metgrid_levels                  = 38,                                   ! number of vertical levels in WPS output
- num_metgrid_soil_levels             = 3,                                    ! number of soil levels or layers in WPS output
+ num_metgrid_levels                  = 27,                                   ! number of vertical levels in WPS output
```
Comment (lukeconibear) [inline WRFotron2.0_CEMAC_ARC4/namelist.wrf.blueprint:41]: ECMWF requires `num_metgrid_levels = 38 `. I think the CEMAC compilation uses this right?

File: WRFotron2.0_CEMAC_ARC4/namelist.wrf.blueprint_201 [hunk 5]
```diff
@@ -16,7 +16,7 @@
  history_interval                    = 60, 60,                               ! history output file interval in minutes (integer only)
  frames_per_outfile                  = 1, 1,                                 ! number of output times bulked into each history file
  restart                             = __ISRESTART__,                        ! whether this run is a restart
- restart_interval                    = 1440,                                 ! restart output file interval in minutes
+ restart_interval                    = 360,                                 ! restart output file interval in minutes
```
Comment (lukeconibear) [inline WRFotron2.0_CEMAC_ARC4/namelist.wrf.blueprint_201]: Could this also be 1440?

**Comment by lukeconibear** [comment]
The only thing that confuses me is that I use NCO version 4.8.1 and I don't have this issue.

**Comment by lukeconibear** [comment]
I've added the NCO version check and variable list check to the non-CEMAC WRFotron2.0.

File: WRFotron2.0_CEMAC_ARC4/namelist.wrf.blueprint (hunk 4 above)
Comment (cemachelen) [reply WRFotron2.0_CEMAC_ARC4/namelist.wrf.blueprint]: I've just copied over the WRFototron2.0 namelist files and replaced. My tests were using GFS data, do you want me to default to ecmwf? I've swapped in config.bash and pre.bash to default to ecmwf.

Comment (lukeconibear) [reply WRFotron2.0_CEMAC_ARC4/namelist.wrf.blueprint]: Okay thanks. 
Most do use ECMWF now, so that's useful.

**Pull request merged**